modules/user_data/templates/start_graphdb.sh.tpl

```diff
@@ -16,126 +16,41 @@ instance_id=$( curl -Ss -H "X-aws-ec2-metadata-token: $imds_token" 169.254.169.2
 availability_zone=$( curl -Ss -H "X-aws-ec2-metadata-token: $imds_token" 169.254.169.254/latest/meta-data/placement/availability-zone )
 volume_id=""
```

Comment:
I don't think `volume_id=""` is used anywhere

Reply:
Done.